### ISSUE TITLE
Running lwm2m on contiki

### ISSUE THREAD

**Issue by tarakanov** [sics-iot/lwm2m-contiki#11]
Hi! I successfully run `oma-lwm2m/standalone` example with DTLS. Now I try to run it on Sensortag platform (cc2650). What should I modify to port it back to Contiki? As I understand this repo has separate coap-transport layer but only for native platform. Is there contiki port of this coap-transport?

I see open [pull request of Contiki separate level](https://github.com/contiki-os/contiki/pull/2156) but it is still open and there is no DTLS support. So could you advise optimal way to port lwm2m/dtls support for non-native contiki?

**Comment by joakimeriksson** [comment]
Yes, we have a separate DTLS implementation in this repository that is significantly refactored and the current support is for "native" or posix - but we aim at adding complete Contiki support soon - so that it should work on any supported Contiki platform (as long as it fits memory). The upstream pull request is likely not the path we are going to take.
